src/Emogrifier.php

```diff
@@ -412,8 +412,7 @@ private function createRawDomDocument($html)
      */
```

Comment:
The description of this method is not accurate anymore. It should be something like:
```
     * Returns the HTML with added document type and Content-Type meta tag if needed.
```
Given this, I am not sure that `unifyHtml` is the best name anymore for this method. What do you think?

Reply:
Good catch! I've updated the descriptions and will repush after we've come to a conclusion about the name.

Reply:
The name should communicate that the method prepares the HTML in a way that it is safe for creating a DOM document from.

What do you think about the name `prepareHtmlForDomConversion`?

Reply:
I like it!

Reply:
Great! Renamed and repushed.